businessRules/donations.js

```diff
@@ -1,15 +1,17 @@
 const { v4 } = require('uuid');
 const { createDonation } = require('../models/donations');
-const { getAllIdeas } = require('../models/ideas');
-const { getIdeas } = require('../utils/storage');
+const {
+    ideaThatMatchesIdeaId,
```

Comment:
It's not clear what this function does. Can you simplify it's name please?

Reply:
Renamed.